### ISSUE TITLE
Remove whitespace helper destroys if statement.

### ISSUE THREAD

**Issue by Aaronepower** [sunng87/handlebars-rust#113]
When the whitespace remover is applied to an else block, the conditional will change, executing neither if the condition is `false`, and both if the condition is `true`.

## Example case
**Handlebars:**
```handlebars
{{#each items}}
    {{#if this}}
        Hello
    {{~else}}
        World
    {{/if}}
{{/each}}
```

**JSON:**
```json
{
    "items": [true, false, true]
}
```

## Output
```
Hello
World
Hello
World
```

**Comment by sunng87** [comment]
Oh, this is a parser bug. Will fix it later.

**Comment by sunng87** [comment]
Released in 0.22
